Source/JavaScriptCore/API/glib/JSCContext.cpp

```diff
@@ -294,7 +294,7 @@ JSValueRef jscContextGArrayToJSArray(JSCContext* context, GPtrArray* gArray, JSV
         else if (JSC_IS_VALUE(item))
             JSObjectSetPropertyAtIndex(priv->jsContext.get(), jsArrayObject, i, jscValueGetJSValue(JSC_VALUE(item)), exception);
         else
-            *exception = toRef(JSC::createTypeError(globalObject, makeString("invalid item type in GPtrArray")));
+            *exception = toRef(JSC::createTypeError(globalObject, makeString("invalid item type in GPtrArray"_s)));
```

Comment:
`makeString` is not needed when it’s just a single argument. This occurs here and a couple other places.

Reply:
Found, fixed.